### DISCUSSION TITLE
Does Auto ISO works as in-place upgrade?

### DISCUSSION THREAD

**Discussion by Jyzark** [AveYo/MediaCreationTool.bat#163] [Q&A] [answered]
I think some system files got corrupted in my Windows 11 PC and I want to do in-place upgrade to fix it.

Can Auto ISO do the job as an in-place upgrade method or should I choose Auto Upgrade?
Also some information, I also used this bat to upgrade from Win 10 to Win 11

**Comment by AveYo** [comment, marked as answer]
You can mount the resulting `Auto ISO` file, and run setup (or preferably `auto.cmd`) and it will do an in-place upgrade as usual.
